### ISSUE TITLE
Addphsp ***ERROR READING HEADER OF PHASE SPACE FILE***

### ISSUE THREAD

**Issue by barghashr** [nrc-cnrc/EGSnrc#914]
Hi,

I have been trying to merge 10 phase space files using "addphsp" utility. It works until after the second phase space file, then gives an error that says "***ERROR READING HEADER OF PHASE SPACE FILE***".

I am not sure what the problem is, please let me know if you could help and if more information is needed.

Thank you,
Reham

![error](https://user-images.githubusercontent.com/111618589/185722205-e888b4bd-5432-400f-b3df-acd720449a74.PNG)

**Comment by rtownson** [comment]
Did this issue get resolved? I feel like I addressed the same question elsewhere and the issue was insufficient disk space.

**Comment by barghashr** [comment]
Correct, it got resolved and the issue was insufficient disk space. 

Sorry for not closing the issue earlier.

Thank you.
